### ISSUE TITLE
When rebasing, and a wrong chrome window gets the request, no easy (if any?) way to recover.

### ISSUE THREAD

**Issue by vaikas** [sigstore/gitsign#110]
**Description**
I have multiple chrome windows open, one for my work, one for personal, and one for say Knative.

When I started to rebase a project that's wired up to use gitsign, it spun up the request to authenticate on one of the wrong windows, which will not work. Usually with other things that require auth, it prints the URL so if a wrong chrome window is spun up, I can cut & paste the URL to the correct window but with gitsign, I don't know how to do that so hit ctrl-c which seems to cause hilarity.

```
vaikas@villes-mbp someproject % git rebase -ff upstream/main
^Cbasing (1/4)
vaikas@villes-mbp someproject % git rebase -ff upstream/main
fatal: It seems that there is already a rebase-merge directory, and
I wonder if you are in the middle of another rebase.  If that is the
case, please try
	git rebase (--continue | --abort | --skip)
If that is not the case, please
	rm -fr ".git/rebase-merge"
and run me again.  I am stopping in case you still have something
valuable there.
```

So, I wonder if it would be useful to print the url where it's trying to use for auth so in case like this I could cut&paste to right chrome window.

**Version**
vaikas@villes-mbp someproject % gitsign --version
unknown

**Comment by imjasonh** [comment]
I agree it's somewhat annoying when you have multiple Chrome profiles open. I find myself copypasting the URL from my personal profile to a new window in my corp profile, and closing the window in my personal profile.

I'm not sure whether `gitsign` can print anything out at all, the way it's invoked by `git`.

**Comment by vaikas** [comment]
Good call as always @imjasonh wish I had thought of it before I hit ctrl-c :) You know how patient I can be :)

**Comment by wlynch** [comment]
I think https://github.com/sigstore/gitsign/pull/87 fixes this, so long as a TTY is available in the session - try updating to v0.2.0!

This is what gisign looks like for me at a version close to head -

```
$ git rebase --continue
Your browser will now be opened to:
https://oauth2.sigstore.dev/auth/auth?access_type=online&client_id=sigstore&code_challenge=...
```

**Comment by vaikas** [comment]
Sure does @wlynch just tried and indeed works out. Thanks much!
